brush-interactive/src/completion.rs

```diff
@@ -72,6 +100,21 @@ fn postprocess_completion_candidate(
                 candidate.push(std::path::MAIN_SEPARATOR);
             }
         }
+
+        if !options.no_autoquote_filenames {
+            let quote_mode = match quote_char {
+                Some(q) => {
+                    if q == '\'' {
+                        escape::QuoteMode::SingleQuote
+                    } else {
+                        escape::QuoteMode::DoubleQuote
+                    }
+                }
+                None => escape::QuoteMode::BackslashEscape,
+            };
+
+            candidate = escape::quote_if_needed(&candidate, quote_mode).to_string();
+        }
     }
     if options.no_autoquote_filenames {
```

Comment:
This debug message was primarily left as a reminder to "handle" autoquote behavior. With your change, it should be safe to remove it; would you mind doing so?

Reply:
Agreed.